stored/src/app/(auth)/login/page.tsx

```diff
@@ -1,3 +1,19 @@
-export default function Login() {
-  return <h1>Login page</h1>;
+import Link from "next/link";
+import { Form as LoginForm } from "./form";
+
+export default function LoginPage() {
+  return (
+    <div className="h-screen-minus-nav flex justify-center items-center bg-slate-100">
+      <div className="sm:shadow-xl px-8 pb-8 pt-12 sm:bg-white rounded-xl space-y-12">
+        <h1 className="font-semibold text-2xl">Login</h1>
+        <LoginForm />
+        <p className="text-center">
+          Need to create an account?{" "}
```

Comment:
remove unnecessary whitespace

Reply:
> remove unnecessary whitespace

or is it for padding? might be nicer to use tailwind for padding instead of spaces